src/components/orders/OrderPriceDisplay/index.tsx

```diff
@@ -15,16 +15,16 @@ import {
 const Wrapper = styled.span`
   display: flex;
   align-items: center;
-  font-size: ${({ theme }): string => theme.fontNormal};
 `
 
 const Icon = styled(FontAwesomeIcon)`
-  background: ${({ theme }): string => theme.textSecondary2}33; /* 33==20% transparency in hex */
+  background: ${({ theme }): string => theme.grey}33; /* 33==20% transparency in hex */
```

Comment:
@alfetopito Actually one case where it's nice to just have our base 6 digit hex as a variable, and just append an HEX opacity to it....

On the other hand, I wonder if we should just store pure RGB values like so:
`green: '0 216 151',

Then use it like so:
`color: rgb(${({ theme }): string => theme.green})`

And a scenario were we set the opacity:
`color: rgb(${({ theme }): string => theme.green} / 10%)`

Maybe too verbose and hardcore? But would allow us to keep opacities outside variables otherwise...

Reply:
> On the other hand, I wonder if we should just store pure RGB values like so:
`green: '0 216 151',
>
> Then use it like so:
color: rgb(${({ theme }): string => theme.green})
>
> And a scenario were we set the opacity:
color: rgb(${({ theme }): string => theme.green} / 10%)

Oh, that's nice. Didn't know it was and option.

To be honest, I thought my solution was hacky (append hex transparency)

I'm in favor of your suggestion, as it gives more flexibility and readability, at the expense of a little convenience (having to wrap the values in `rgb()`)
@anxolin @W3stside WDYT?

Reply:
The only inefficiency in doing so, is that we'd probably benefit from storing opacity variants also as a variable to have uniform styling? So keep it simple as we have I mean.